acme/localStore_unix.go

```diff
@@ -7,19 +7,22 @@ import (
 	"os"
 )
 
-// Check file permissions
-func checkPermissions(name string) error {
+// Check file permissions and content size
+func checkFile(name string) (bool, error) {
 	f, err := os.Open(name)
 	if err != nil {
-		return err
+		return false, err
 	}
 	defer f.Close()
+
 	fi, err := f.Stat()
 	if err != nil {
-		return err
+		return false, err
 	}
+
 	if fi.Mode().Perm()&0077 != 0 {
-		return fmt.Errorf("permissions %o for %s are too open, please use 600", fi.Mode().Perm(), name)
+		return false, fmt.Errorf("permissions %o for %s are too open, please use 600", fi.Mode().Perm(), name)
```

Comment:
Are these permissions something that we could push to a variable? so that at a later date, it could be customized if needed? I know users in the past have complained about this...

Reply:
@dtomcej the `600` right cannot be customized, it's seem to be needed by Let's Encrypt.